mobile/lib/features/trade/trade_dialog.dart

```diff
@@ -125,3 +164,47 @@ Future<void> shareTweet(PositionAction action) async {
     await Share.share(shareText);
   }
 }
+
+class ClickableHelpText extends StatelessWidget {
+  const ClickableHelpText({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return RichText(
+      text: TextSpan(
+        text: 'Please help us fix this issue and join our telegram group: ',
+        style: DefaultTextStyle.of(context).style,
+        children: [
+          TextSpan(
+            text: 'https://t.me/get10101',
+            style: const TextStyle(
+              color: Colors.blue,
+              decoration: TextDecoration.underline,
+            ),
+            recognizer: TapGestureRecognizer()
+              ..onTap = () async {
+                final httpsUri = Uri(scheme: 'https', host: 't.me', path: 'get10101');
+                if (await canLaunchUrl(httpsUri)) {
```

Comment:
❓ i remember that this url launcher can go external or within the app. I think the later is the default. But can you clarify which one it is here?

FYI: I'd prefer to open an external app than opening a browser in app.

Reply:
It would have used the platform default.

```
///   - [LaunchMode.platformDefault] is supported on all platforms:
///     - On iOS and Android, this treats web URLs as
///       [LaunchMode.inAppWebView], and all other URLs as
///       [LaunchMode.externalApplication].
///     - On Windows, macOS, and Linux this behaves like
///       [LaunchMode.externalApplication].
///     - On web, this uses `webOnlyWindowName` for web URLs, and behaves like
///       [LaunchMode.externalApplication] for any other content.
```

I'll fix this.